conduct/code_of_conduct.md

```diff
@@ -0,0 +1,123 @@
+person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you
+report it by emailing
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) or by filling out
+[this form](https://goo.gl/forms/sJzOIie3zde9M71T2). For more details
+please see our Reporting Guidelines.
+
+By embracing the following principles, guidelines and actions to follow
+or avoid, you will help us make Jupyter a welcoming and productive
+community.
+If that doesn't answer your questions, feel free to contact us at
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org).
+
+
+1. **Be friendly and patient**.
+
+2. **Be welcoming**. We strive to be a community that welcomes and supports
+   people of all backgrounds and identities. This includes, but is not limited
+   to, members of any race, ethnicity, culture, national origin, color,
+   immigration status, social and economic class, educational level, sex, sexual
+   orientation, gender identity and expression, age, physical appearance, family
+   status, political belief, technological or professional choices, academic
+   discipline, religion, mental ability, and physical ability.
+
+3. **Be considerate**. Your work will be used by other people, and you in turn
+   will depend on the work of others. Any decision you take will affect users
+   and colleagues, and you should take those consequences into account when
+   making decisions. Remember that we're a world-wide community. You may be
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+   * Violent threats or violent language directed against another person
+   * Discriminatory jokes and language
+   * Posting sexually explicit or violent material
+   * Posting (or threatening to post) other people's personally identifying
+     information ("doxing")
+   * Personal insults, especially those using racist or sexist terms
+   * Unwelcome sexual attention
+   * Advocating for, or encouraging, any of the above behavior
+   * Repeated harassment of others. In general, if someone asks you to stop,
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time.
+   They are probably not purposefully ignoring issues, refusing to engage in
+   discussion, avoiding features, etc. but often just unavailable.
```

Comment:
Minor wording suggestion: ~etc. but often just unavailable~ ➡️  "etc., rather they are usually tending to other aspects of their lives."

I don't like the word "just" because it lends a trivial air to things that is usually best avoided.